client/notebooks/OpenDP_Polars_Demo_BUG.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
Maybe this does not need to be pushed and can be kept in a private repo or .gitignore ? 
Or again, if it will be useful afterwards, for me it is okay to merge in the polars feature branch and then use it for debugging and then remove it before merge into develop.

Same comment for opendp_polars_bug.ipynb

Reply:
yes, i think those are notebooks related to the issue we found with Damien (https://github.com/opendp/opendp/issues/1908) , but i agree it should not be kept on develop.